### DISCUSSION TITLE
Sync external and built-in brightness when changed through custom shortcuts

### DISCUSSION THREAD

**Discussion by champagnealexandre** [MonitorControl/MonitorControl#860] [General]
Hi!

Thanks for this amazing software!

I have one question: I have checked "sync brightness changes from built-in and apple displays". This works great adjusting the brightness through the Touch bar slider, but when setting the brightness through the custom shortcut I have assigned (I hate the touch bar), brightness only changes on one display.

Is there any way to sync the two, even when using custom keyboard shortcuts?

Thanks!

**Comment by waydabber** [comment]
Yes, change the `Screen to control` option to `Change all screens`.

<img width="798" alt="Screen Shot 2021-12-23 at 21 00 15" src="https://user-images.githubusercontent.com/37590873/147286858-186038c4-d83e-44ac-a5f5-77d7eb898f1c.png">

**Reply by champagnealexandre** [reply]
Oh I had missed that one, wonderful !

Thanks!
